src/components/RemediationDetailsTable.js

```diff
@@ -53,10 +53,10 @@ function issueDescriptionCell (issue) {
 
 function needsRebootCell (needsReboot) {
     if (needsReboot) {
-        return <CheckIcon/>;
+        return <CheckCircleIcon className="ins-c-remediations-reboot-check-circle"/>;
```

Comment:
can we add an `aria-label="reboot required"` to this? I noticed it wasn't there initially, but this should be super helpful for screen readers.

Reply:
added it